src/processor/operator/table_function_call.cpp

```diff
@@ -54,6 +54,9 @@ void TableFunctionCall::initGlobalStateInternal(ExecutionContext*) {
 bool TableFunctionCall::getNextTuplesInternal(ExecutionContext*) {
     localState.funcOutput.dataChunk.state->getSelVectorUnsafe().setSelSize(0);
     localState.funcOutput.dataChunk.resetAuxiliaryBuffer();
+    for (auto i = 0u; i < localState.funcOutput.dataChunk.getNumValueVectors(); i++) {
+        localState.funcOutput.dataChunk.getValueVector(i)->setAllNonNull();
```

Comment:
Isn't this suppose to be maintained in each tableFunction? i.e. whoever is changing null mask should reset its own.